hive-tables-creation.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+
+client = boto3.client('glue')
+response = client.create_table(
+    DatabaseName='analytical_dataset_generation',
+    TableInput={
+        'Name': 'core_contract_adg',
```

Comment:
I don't like the `adg` suffix here; they're already namespaced by being in the `analytical_dataset_generation` database.

Reply:
we  create two hive tables one to access hbase and other  for analytical env to access .so I tried to differentiate between them with the suffix